README.md

```diff
@@ -44,6 +44,8 @@ further to fit a particular use case.
   Generic API proxy to provide Create, Read and Delete operations for KVMs
 - [Dutch Healthcare Reference](references/dutch-healthcare) -
   An API reference to accelerate implementation of Dutch Healthcare standards.
+- [XML & JSON Threat Protection](references/threat-protect) -
+  A reference to generate threats and test XML and JSON threat protection
```

Comment:
Maybe make it sound less like a hacker tool if you emphasise the threat protection part like.

```
A reference for protecting API proxies against XML and JSON threats
```

or something

Reply:
Done